pr-deploy/src/github.ts

```diff
@@ -29,11 +29,11 @@ const owner = process.env.GH_OWNER;
 const repo = process.env.GH_REPO;
 
 export class PullRequest {
-  private github: GitHubAPI;
+  private github: Octokit;
   private headSha: string;
 
   constructor(github: GitHubAPI, headSha: string) {
-    this.github = github;
+    this.github = github as unknown as Octokit;
```

Comment:
I remember dealing with this Probot/Octokit type incompatibility for Invite Bot. Supremely obnoxious :angry:

Reply:
Haha yeah what the heck!